torchtune/data/_instruct_templates.py

```diff
@@ -37,6 +39,7 @@ def format(
         pass
 
 
+@deprecated()
```

Comment:
Why no message for this one?

Reply:
re our discussion on the alpaca PR, the template will just be absorbed into the message transform. this class will be removed anyway instead of deprecated